packages/Button/src/LinkButton.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import Button from "./Button";
+import buttonStyles from "./Button.styles";
+
+const LinkPropTypes = {
+  children: PropTypes.node.isRequired,
+  kind: PropTypes.oneOf(Button.Kinds.ALL),
+  size: PropTypes.oneOf(ShirtSizes.MEDIUM),
+  href: PropTypes.string,
+  text: PropTypes.string,
+  isOpenNewTab: PropTypes.bool,
+};
+
+const LinkDefaultProps = {
+  kind: Button.Kinds.DEFAULT,
+  size: ShirtSizes.MEDIUM,
+  href: null,
+  text: "Link",
+  isOpenNewTab: true,
+};
+
+const LinkButton = React.forwardRef((props, ref) => {
+  const buttonProps = {
+    children: null,
+  };
+  console.log(props);
+  if (props.isOpenNewTab) {
```

Comment:
I'd create a separate variable that can hold

```
target="_blank"
rel="noopener noreferrer"
```

based on if there is a `prop.isOpenNewTab` or not. Then we don't need to render two different `<a>`. We just render one.

For example
`<a {...isOpenNewTabProps} {...buttonProps} etc....>`

Reply:
done